server/boot/user.js

```diff
@@ -631,4 +645,51 @@ module.exports = function(app) {
       return res.render('account/forgot');
     });
   }
+
+  function getReportUserProfile(req, res) {
+    const username = req.params.username.toLowerCase();
+    return res.render('account/report-profile', {
+      title: 'Report User',
+      username
+    });
+  }
+
+  function postReportUserProfile(req, res) {
+    const username = req.params.username.toLowerCase();
+    const report = req.body.reportDescription;
```

Comment:
Both the `reportDescription` and username should be sanitized just in case. We don't want malicious scripts to be sent to out email